test/extended/operators/check_crd_status_test.go

```diff
@@ -0,0 +1,102 @@
+package operators
+
+import (
+	"fmt"
+	"os"
+	"strings"
+	"testing"
+
+	exutil "github.com/openshift/origin/test/extended/util"
+	v1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
+	apiextensionsclientset "k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset"
+)
+
+// check_crd_subresource_status gets the list of CustomeResourceDefinitions and
+// checks if the subresource.status stanza is present.  It also does a count.
+// This is meant to be a test you can run outside of CI.
+//
```

Comment:
We don't want to commit this second variant correct?

It should be possible to run a specific ginko test, assuming you've oc login'd to the correct cluster under test, 

```
$ with make && ./openshift-tests run-test "[sig-arch] ClusterOperators should define valid related objects [Suite:openshift/conformance/parallel]"
```

Reply:
I tried the command above (but just built `openshift-tests` separately and ran it) and it works.  So I remove it.  Thanks for the tip.